ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/screens/splashscreen/splash.kt

```diff
@@ -0,0 +1,32 @@
+package at.ict4d.ict4dnews.screens.splashscreen
+
+import android.content.Intent
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import at.ict4d.ict4dnews.R
+import at.ict4d.ict4dnews.screens.MainNavigationActivity
+
+class splash : AppCompatActivity() {
+    private var mDelayHandler: Handler? = null
```

Comment:
We normally don't use the mVariableName pattern, please just call it `delayHandler`
And in Kotlin you can also do:
`private val mDelayHandler = Handler()`